tools/openapi-schema/ansible-ai-connect-service.yaml

```diff
@@ -664,6 +664,14 @@ components:
         * `0` - ACCEPTED
         * `1` - REJECTED
         * `2` - IGNORED
+    SentimentEnum:
+      enum:
+      - '0'
+      - '1'
+      type: string
+      description: |-
+        * `0` - NOT_LIKE
+        * `1` - LIKE
```

Comment:
We define <thumbs_down> = 1, <thumbs_up> = 0.  I think these two lines need to be swapped.

Reply:
gosh! yeah good catch, sorry @TamiTakamiya , fixed.